public/components/Flyout/Flyout.js

```diff
@@ -45,7 +45,11 @@ const Flyout = ({ flyout, onClose }) => {
   } = flyoutData;
 
   const flyoutHeader = header && <EuiFlyoutHeader {...headerProps}>{header}</EuiFlyoutHeader>;
-  const flyoutBody = body && <EuiFlyoutBody {...bodyProps}>{body}</EuiFlyoutBody>;
+  const flyoutBody = body && (
+    <EuiFlyoutBody onClose={onClose} {...bodyProps}>
```

Comment:
Why add `onClose` for `EuiFlyoutBody` ? Line 56 already add `onClose` for `EuiFlyout`

Reply:
fixed